app/app/crashes/page.tsx

```diff
@@ -17,6 +18,7 @@ export default function Crashes() {
             columns={crashesListViewColumns}
             initialQueryConfig={crashesListViewQueryConfig}
             localStorageKey={localStorageKey}
+            schema={crashesListSchema}
```

Comment:
here we are passing in our zod schema, whose type is also referenced by the `crashesListViewColumns` when it is defined in `configs/crashesListViewColumns.tsx`. so the zod-generated type is going to provide all the typing for this table component, including data fetching. See `TableWrapper` for more notes on this...